app/product/[handle]/page.tsx

```diff
@@ -2,16 +2,14 @@ import type { Metadata } from 'next';
 import { notFound } from 'next/navigation';
 import { Suspense } from 'react';
 
-import Grid from 'components/grid';
+import { GridTileImage } from 'components/grid/tile';
 import Footer from 'components/layout/footer';
-import ProductGridItems from 'components/layout/product-grid-items';
```

Comment:
Is this component used anywhere else? Can it be deleted?

Reply:
We use it in the search page

Reply:
👍